src/views/shared/button/button.view.jsx

```diff
@@ -6,22 +6,19 @@ import useButtonStyles from "./button.styles";
 
 function Button({ Icon, text, className, disabled, onClick }) {
   const classes = useButtonStyles({ rounded: !text });
-  const isClickable = onClick !== undefined;
```

Comment:
Why stop supporting this behaviour?

Reply:
And where is this button used?

Reply:
Because I think that a Button component must have an onClick prop and should be clickable.
![image](https://user-images.githubusercontent.com/19190248/134367508-dab0a6cd-defc-4aae-be85-48f90c95127c.png)
![image](https://user-images.githubusercontent.com/19190248/134367564-33f005b7-6a92-4a68-98a8-21b2c379f7e3.png)

Reply:
If all instances are clickable and it doesn't break anything, then great!

Reply:
Yes, all components are passing that prop